CHANGELOG.md

```
@@ -1,3 +1,27 @@
+## 0.8.0 (2025-09-27)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)